packages/clerk-js/src/ui/portal/index.tsx

```diff
@@ -19,7 +19,7 @@ export default class Portal<CtxType extends AvailableComponentCtx> extends React
 
     const el = (
       <ComponentContext.Provider value={{ componentName: componentName, ...props } as CtxType}>
-        {React.createElement(component, props)}
+        <Suspense fallback={''}>{React.createElement(component, props)}</Suspense>
```

Comment:
❓ @panteliselef What's the UX until the components load? Is there a flickering? Do we need a skeleton or spinner? Can you please share a screencast?